pkgs/servers/uwsgi/default.nix

```diff
@@ -74,7 +79,7 @@ stdenv.mkDerivation rec {
   buildPhase = ''
     mkdir -p $pluginDir
     python3 uwsgiconfig.py --build nixos
-    ${lib.concatMapStringsSep ";" (x: "${x.interpreter or "python3"} uwsgiconfig.py --plugin ${x.path} nixos ${x.name}") needed}
+    ${lib.concatMapStringsSep ";" (x: "${x.preBuild or ""} ${x.interpreter or "python3"} uwsgiconfig.py --plugin ${x.path} nixos ${x.name}") needed}
```

Comment:
Add a semicolon after `preBuild` here so that we wouldn't need to add it in every `preBuild` by hand. You can fallback to `true` so that if `preBuild` is not set it evaluates to `true;`.